### DISCUSSION TITLE
cast call 0xaECf7C5F07d4274F72C0e21fC6c9D6e40b19a9A6 "getRecentWinner()" --rpc-url $SEPOLIA_RPC_URL 0x0000000000000000000000000000000000000000000000000000000000000000

### DISCUSSION THREAD

**Discussion by yusuphoo** [Cyfrin/foundry-full-course-cu#1423] [Q&A]
No address returned when `cast call 0xaECf7C5F07d4274F72C0e21fC6c9D6e40b19a9A6 "getRecentWinner()" --rpc-url $SEPOLIA_RPC_URL`
```solidity
cast call 0xaECf7C5F07d4274F72C0e21fC6c9D6e40b19a9A6 "getRecentWinner()" --rpc-url $SEPOLIA_RPC_URL
0x0000000000000000000000000000000000000000000000000000000000000000
```

**Comment by EngrPips** [comment]
The above context shows that you got back address(0) as the returned value, which is always the initial value of a type of address in solidity, so i think you haven't actually commenced the game at all and have a registered winner and thats why you get back address(0) when you did make that call

**Reply by yusuphoo** [reply]
I entered `0.01` in my `enterRaffle` function and the `getNumberOfPlayers` shows `1 uint256` but the `getrecentWinner` shows `0x0000000000000000000000000000000000000000 address`
